### PR TITLE
15 create a register component

### PR DESCRIPTION
**Pull request by raj-sharma1**
Register component and test working.

### CODE REVIEW

File: client/components/App.jsx [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
 const App = () => {
   return (
     <Router>
       <Switch>
+<<<<<<< HEAD
+        <Route exact path='/register' component={Register} />
+=======
       <Route path ='/' component={Header} />
         <h1>MOA development has begun!</h1>      
+>>>>>>> c03a930193e9bafa776d9a4c0a7e6315299893e1
```
Comment (emilyparkes) [inline client/components/App.jsx]: oops looks like a pull from dev left some code here, please remove!

File: client/components/Register.jsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+render () {
+  return (
+    <>
+    <div className='register'>
+      <input name='first_name'
+        placeholder='First Name'
+        value={this.state.first_name}
+        onChange={e => this.onChange(e)}
+      />
+      <br/>
```
Comment (emilyparkes) [inline client/components/Register.jsx]: Please remove <br> tags as this will be solved when we do styling

File: client/components/Register.jsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+
+class Register extends React.Component {
+state = {
+  first_name: '',
```
Comment (emilyparkes) [inline client/components/Register.jsx]: Can we use camelCase?

File: tests/client/components/Register.test.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+// test('test harness is setup correctly', () => {
+//   expect(true).toBe(true)
+// })
+
+// test('/ register return a status of 200', () => {
+//   request(server)
+//     .get('/register')
+//     .expect(200)
+//     .end((err, res) => {
```
Comment (emilyparkes) [inline tests/client/components/Register.test.js]: No comments please!

**Review by emilyparkes — CHANGES_REQUESTED** [review]

**Comment by raj-sharma1** [comment]
@emilyparkes changes have been done, can you please review it again.

**Comment by raj-sharma1** [comment]
changes made on package json

closes #15

**Review by emilyparkes — APPROVED** [review]

**Pull request merged**